cmake/modules/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Since we just use MLIR's cmake functions (add_mlir_library, etc.),
+# MLIR_EXPORTS would have the necessary targets to export.
+get_property(STABLEHLO_EXPORTS GLOBAL PROPERTY MLIR_EXPORTS)
+
+set(STABLEHLO_CONFIG_CMAKE_DIR "${CMAKE_BINARY_DIR}/lib/cmake/stablehlo")
+
+export(TARGETS ${STABLEHLO_EXPORTS}
+  NAMESPACE Stablehlo::
+  FILE "${STABLEHLO_CONFIG_CMAKE_DIR}/StablehloTargets.cmake"
+)
+
+# Set variables and configure StablehloConfig.cmake for the _build_ tree.
+set(STABLEHLO_CONFIG_INCLUDE_DIRS
+  "${STABLEHLO_SOURCE_DIR}"
+  "${STABLEHLO_BINARY_DIR}"
+)
+
+configure_file(
+  "${CMAKE_CURRENT_SOURCE_DIR}/StablehloConfig.cmake.in"
+  "${STABLEHLO_CONFIG_CMAKE_DIR}/StablehloConfig.cmake"
+  @ONLY
+)
+
+export(EXPORT MLIRTargets
+  NAMESPACE Stablehlo::
+  FILE "${PROJECT_BINARY_DIR}/StablehloTargets.cmake"
+)
+
```

Comment:
The targets were already exported on lines 7--10, so this export is not needed and results in an error about duplicate imports in packages using the exported targets.